packages/cli/src/util/colorizer.ts

```diff
@@ -0,0 +1,23 @@
+import { mapValues, isArray } from 'lodash';
+import { Dictionary } from '@stoplight/types';
+
+export type ValuesTransformer = (values: Dictionary<unknown>) => Dictionary<string | string[]>;
+
+export const PRE_PARAM_VALUE_TAG = '~pre~';
+export const POST_PARAM_VALUE_TAG = '~post~';
+
+export const transformPathParamsValues = ((
+  taggedParamsValues: RegExp,
+  path: string,
+  transform: (aString: string) => string
+): string => {
+  return path.replace(taggedParamsValues, transform('$2'));
+}).bind({}, new RegExp(`(${PRE_PARAM_VALUE_TAG},)(.*?)(,${POST_PARAM_VALUE_TAG})`, 'gm'));
```

Comment:
Hmm maybe instead of binding you can just create the RegEx ahead of the time as a constant and use it in there — this should improve the readability.